src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Numerics;
+
+namespace System.Runtime.Intrinsics.Arm
+{
+    /// <summary>
+    /// This class provides access to the ARM SVE hardware instructions via intrinsics
+    /// </summary>
+    [Intrinsic]
+    [CLSCompliant(false)]
+    public abstract class Sve : AdvSimd
+    {
+        internal Sve() { }
+
+        public static new bool IsSupported { get => IsSupported; }
```

Comment:
we will have to make sure to return `false` for Mono

Reply:
@fanyang-mono - do you know?

Reply:
> One way of doing it is to add a new element to this array

aren't these the entries of things that are supported? so probably no SVE entry is needed in that array?

Reply:
When you specify `unsupported`, `SN_get_IsSupported` will return false. So it is needed.

Reply:
Thanks! I can also see some examples of unsupported in `supported_x86_intrinsics`.